src/utils/TranslationUtils.js

```diff
@@ -38,25 +42,37 @@ const addTranslationParametersGroupsLabels = (parametersGroups) => {
 
 const addTranslationParametersLabels = (parameters) => {
   const resources = {};
+  const _addResource = (lang, key, value) => {
+    if (resources[lang] == null) resources[lang] = {};
```

Comment:
You can use the nullish coalescing operator : `resources[lang] = resources[lang] ?? {}`